.travis.yml

```diff
@@ -4,3 +4,7 @@ node_js:
   - '10'
   - '12'
 script: ./script/travis_run_js_tests.sh
+ 
+after_success:
+  - chmod ugo+x ./script/storybook-deploy.sh
```

Comment:
I don't think it is needed, you can run `chmod +x ./script/storybook-deploy.sh` in your local env once and then push it to github.